src/ui/settings-tools.ts

```
@@ -0,0 +1,210 @@
+		}
+	}
+}
+
+async function showYoloConfirmation(app: App): Promise<boolean> {
+	return new Promise(async (resolve) => {
+		const { YoloConfirmationModal } = await import('./yolo-confirmation-modal');
+		const modal = new YoloConfirmationModal(app, (confirmed: boolean) => {
+			resolve(confirmed);
+		});
+		modal.open();
+	});
```

Biome (2.4.10)

[error] Promise executor functions should not be `async`.

(lint/suspicious/noAsyncPromiseExecutor)